modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/exp/ExpressionFactoryImpl.java

```diff
@@ -551,6 +553,18 @@ private String digest(List<RexNode> nodes, RelDataType type, boolean biParam) {
                 continue;
             }
 
+            if (node instanceof RexCall) {
+                RexCall call = (RexCall) node;
+                if (!call.operands.isEmpty()) {
```

Comment:
why we don`t want to rely on RexCall#toString -> RexCall#computeDigest(boolean) ?

Reply:
because it doesn't works